packages/search-ui/src/SearchDriver.js

```diff
@@ -178,28 +175,11 @@ export default class SearchDriver {
 
     if (isLoading && !ignoreIsLoadingCheck) return;
 
-    const searchOptions = {
-      disjunctiveFacets: this.disjunctiveFacets,
-      disjunctiveFacetsAnalyticsTags: this.disjunctiveFacetsAnalyticsTags,
-      facets: removeConditionalFacets(
-        this.facets,
-        this.conditionalFacets,
-        filters
-      ),
-      filters: adaptFilters(filters),
-      page: {
-        current,
-        size: resultsPerPage
-      },
-      result_fields: this.result_fields,
-      search_fields: this.search_fields
-    };
-
-    if (sortField && sortDirection) {
-      searchOptions.sort = {
-        [sortField]: sortDirection
-      };
-    }
+    // facets: removeConditionalFacets(
```

Comment:
👀

Reply:
Removed this, it was moved elsewhere and the test is still passing 👍